### ISSUE TITLE
Could you provide a minimal inference script?

### ISSUE THREAD

**Issue by SI-Abid** [SamsungLabs/hifi_plusplus#4]
Hi, @AndreevP 
I have tried to run inference using the given weights on the voiceband-demand test set. After which I ran the wave2vec mos. But the score I got didn't match what I shown in the paper. I am assuming that I might have missed any steps during Inference. I am sharing my script for inference below. I would be very much grateful if you could provide me with a inference script or steps that should be followed.

```python
import librosa
from tqdm import tqdm
from models.hifi import A2AHiFiPlusGeneratorV2

from glob import glob
import os
import torch
import torchaudio

import utils


model = A2AHiFiPlusGeneratorV2()

# Load the model weights
model.load_state_dict(torch.load("weights/se.pth"))
model.eval()
if torch.cuda.is_available():
    model = model.cuda()


def inference(input_dir, output_dir):
    if not os.path.exists(output_dir):
        os.makedirs(output_dir)

    input_files = glob(os.path.join(input_dir, "*.wav"))
    for input_file in tqdm(input_files):
        filename = os.path.basename(input_file)
        output_file = os.path.join(output_dir, filename)
        # print(f"Processing {input_file} -> {output_file}")

        # Load the audio file
        x, sr = torchaudio.load(input_file)
        if sr != 16000:
            resample = torchaudio.transforms.Resample(orig_freq=sr, new_freq=16000)
            x = resample(x)
            sr = 16000
        # pad_size = utils.closest_power_of_two(x.shape[-1]) - x.shape[-1]
        x = torch.tensor(x)
        if torch.cuda.is_available():
            x = x.cuda()

        # audio = torch.nn.functional.pad(x, (0, pad_size))
        audio = x.reshape(1, 1, -1)
        # Inference
        with torch.no_grad():
            output_audio = model(audio).squeeze(1)

        # Save the output audio
        torchaudio.save(output_file, output_audio.cpu(), sample_rate=sr)


if __name__ == "__main__":
    input_dir = "/home/saiham/Data/VoiceBank-16k/test/noisy"
    output_dir = "/home/saiham/Data/VoiceBank-16k/test/hifi3"
    inference(input_dir, output_dir)
    # Example usage

```
